platform/kube/inject/testdata/auth.cert-dir.yaml.injected

```diff
@@ -49,10 +49,16 @@ spec:
         securityContext:
           runAsUser: 1337
         volumeMounts:
+        - mountPath: /etc/istio
```

Comment:
once again, I think /home/istio is a bad idea. Expecting people to create a /home for a system user is not ideal. If anything, people would prefer /var/run or /var/lib/ . And its best to keep this configurable.. If we have debian packaging, then its easy to automate installation of istio user, and create /var/lib/istio path, that is world readable and writable only by istio.